packages/eslint-plugin-orbit-components/src/rules/arrowFunctionsCount.ts

```diff
@@ -0,0 +1,49 @@
+import * as t from "@babel/types";
+import type { Rule } from "eslint";
+
+const LIMIT = 5;
+export const ERROR =
+  "You are using too many arrow function expressions. You can desctructure all props in single function and pass it down via css helper function";
```

Comment:
You are using too many arrow function expressions in one single styled component. This can negatively impact performance. Consider using one single function to desctructure all props and return one single css helper function if possible.